features/signals_railway_signals.yaml

```diff
@@ -4707,114 +4727,266 @@ features:
       - { tag: 'railway:signal:main:form', value: 'light' }
       - { tag: 'railway:signal:main:substitute_signal', value: 'IT:AVV' }
 
+     - description: Indicatori di partenza (Avvio)
+     country: IT
+     icon: { default: 'it/PAR-AVV' }
+     tags:
+       - { tag: 'railway:signal:departure', value: 'IT:PAR' }
+       - { tag: 'railway:signal:departure:form', value: 'light' }
+       - { tag: 'railway:signal:departure:substitute_signal', value: 'IT:AVV' }
+
+#-------------
+# ... to there. with the same result. ----
+
+  - description: Indicatori di partenza
+    country: IT
+    icon: { default: 'it/PAR' }
+    tags:
+      - { tag: 'railway:signal:departure', value: 'IT:PAR' }
+      - { tag: 'railway:signal:departure:form', value: 'light' }
+
   - description: 1ª categoria (1 light)
     country: IT
     icon:
       match: 'railway:signal:main:states'
       cases:
+        - { any: ['(Y)','Y'], value: 'it/main-1v-Y' }
         - { exact: 'G', value: 'it/main-1v-G' }
       default: 'it/main-1v-R'
     tags:
       - { tag: 'railway:signal:main', value: 'IT:1V' }
       - { tag: 'railway:signal:main:form', value: 'light' }
 
+  - description: 1ª categoria quadro (1 light)
```

Comment:
This signal is more specific than the one above, so it should be put above the `IT:1V` without the square shape matching. Otherwise the one above will match first, and this one will never match. Matching happens from top to bottom in this file, per type of signal.